ote/ote/utils/runners.py

```diff
@@ -79,3 +79,11 @@ def run_with_termination(cmd):
         stderr = stderr.decode('utf-8')
         print(stderr, end='')
         sys.stdout.flush()
+
+    time.sleep(1)
+    err = f'Out of memory: Killed process {process.pid}'
```

Comment:
Seems like sleep is not required in since dmesg reads directly from the kernel's message buffer.